### PR TITLE
fix(differences): What to do if you need pre-EIP-155

### PR DESCRIPTION
**Pull request by qbzzt**

### CODE REVIEW

File: src/docs/developers/build/differences.md [hunk 1]
```diff
@@ -146,8 +146,15 @@ See the full list of [custom JSON-RPC methods](./json-rpc.md) for more informati
 
 [Pre-EIP-155](https://eips.ethereum.org/EIPS/eip-155) transactions do not have a chain ID, which means a transaction on one Ethereum blockchain can be replayed on others.
 This is a security risk.
-Starting in November 2022, pre-EIP-155 transactions are no longer supported on Optimism.
+Starting in November 2022, pre-EIP-155 transactions are no longer supported on Optimism through Alchemy.
```
Comment (mslipper) [inline src/docs/developers/build/differences.md:149]: This should read "on the public endpoint or through Alchemy" to make it super clear which RPCs won't support these transactions.

Comment (qbzzt) [reply src/docs/developers/build/differences.md:149]: I don't understand this. If I submit a pre-EIP-155 transaction to an Infura endpoint, doesn't it get relayed to our sequencer, which will then reject it? Or is it that the sequencer itself accepts pre-EIP-155 transactions, but the public endpoint is a proxy which doesn't?

File: src/docs/developers/build/differences.md [hunk 2]
```diff
@@ -146,8 +146,15 @@ See the full list of [custom JSON-RPC methods](./json-rpc.md) for more informati
...
 [Pre-EIP-155](https://eips.ethereum.org/EIPS/eip-155) transactions do not have a chain ID, which means a transaction on one Ethereum blockchain can be replayed on others.
 This is a security risk.
-Starting in November 2022, pre-EIP-155 transactions are no longer supported on Optimism.
+Starting in November 2022, pre-EIP-155 transactions are no longer supported on Optimism using the public endpoint or through Alchemy.
+
+::: warning Pre-EIP-155 transactions are dangerous
 
+It is highly recommended not to use pre-eip-155 transaction.
+But if you absolutely must use them, [Infura](../../useful-tools/providers.md#infura) and [QuickNode](../../useful-tools/providers.md#quicknode) still support them.
+Just be careful.
```
Comment (Nicca42) [inline src/docs/developers/build/differences.md:155]: Could we add something about why these pre EIP-155 TXs are dangerous?

File: src/docs/developers/build/differences.md [hunk 3]
```diff
@@ -145,9 +145,20 @@ See the full list of [custom JSON-RPC methods](./json-rpc.md) for more informati
 ### Pre-EIP-155 support
 
 [Pre-EIP-155](https://eips.ethereum.org/EIPS/eip-155) transactions do not have a chain ID, which means a transaction on one Ethereum blockchain can be replayed on others.
-This is a security risk.
-Starting in November 2022, pre-EIP-155 transactions are no longer supported on Optimism.
+This is a security risk, because transactions that are legitimate on one chain could be a security risk on another.
+For example, you might agree to send me 1 ETH on Georli (chain ID 5) to help me test my contracts.
```
Comment (Nicca42) [inline src/docs/developers/build/differences.md:149]:
```suggestion
For example, you might agree to send me 1 ETH on Goerli (chain ID 5) to help me test my contracts.
```

**Review by Nicca42 — CHANGES_REQUESTED** [review]

**Comment by qbzzt** [comment]
When you have an obvious change, such as Georli -> Goerli, you can just do it instead of having me approve it. That way it's not waiting.

**Review by Nicca42 — APPROVED** [review]

**Pull request merged**